### PR TITLE
Doc: Add details

### PR DESCRIPTION
**Pull request by s-pace**
CC feedback from users

### CODE REVIEW

File: docs/src/dropdown.md [hunk 1]
```diff
@@ -35,6 +35,9 @@ config.
 </script>
 ```
 
+This snippet needs to be integrated to every pages that should integrate the
```
Comment (Haroenv) [inline docs/src/dropdown.md]:
```suggestion
This snippet needs to be integrated to every page that integrates the dropdown UI
```

File: docs/src/dropdown.md [hunk 2]
```diff
@@ -35,6 +35,9 @@ config.
...
+dropdown UI.
```
Comment (Haroenv) [inline docs/src/dropdown.md]:
```suggestion
```

removed if first suggestion is accepted

File: docs/src/faq.md [hunk 3]
```diff
@@ -129,6 +129,21 @@ It creates a brand new temporary index to populate the data scraped from your
 website. Once the crawl is successfully achieved, this temporary index overwites
 the old index defined in your configuration with the key `index_name`.
 
+## Why my pages is not indexed or can not be retrieved?
+
+We are crawling your website according to your configuration. It might happen
```
Comment (Haroenv) [inline docs/src/faq.md]:
```suggestion
We are scraping your website according to your configuration. It might happen
```

File: docs/src/faq.md [hunk 4]
```diff
@@ -129,6 +129,21 @@ It creates a brand new temporary index to populate the data scraped from your
...
+reasons for that:
+
+- Makes sure you are not filtering on the search by wrongly using
+  `facetFilters`. [See here for more details][11].
+- Make sure the page missing is referenced from an other indexed page thanks to
+  a hyperlink tag `<a/>`.
+- Make sure you are [providing a compliant sitemap from the configuration][12]
+  and that this page is referenced by it.
+
+If none of the previous points helps, you can contact our support.
```
Comment (Haroenv) [inline docs/src/faq.md]:
```suggestion
If none of the previous points help, you can contact our support.
```

File: docs/src/faq.md [hunk 5]
```diff
@@ -129,6 +129,21 @@ It creates a brand new temporary index to populate the data scraped from your
 website. Once the crawl is successfully achieved, this temporary index overwites
 the old index defined in your configuration with the key `index_name`.
 
+## Why my pages is not indexed or can not be retrieved?
```
Comment (francoischalifour) [inline docs/src/faq.md]:
```suggestion
## Why aren't my pages indexed?
```

File: docs/src/faq.md [hunk 6]
```diff
@@ -129,6 +129,21 @@ It creates a brand new temporary index to populate the data scraped from your
...
+
+We are crawling your website according to your configuration. It might happen
+that some pages might be missing from the search. There is several possible
```
Comment (francoischalifour) [inline docs/src/faq.md]:
```suggestion
that some pages are missing from the search. There are several possible
```

File: docs/src/faq.md (hunk 4 above)
Comment (francoischalifour) [inline docs/src/faq.md]: Add link to support?

File: docs/src/dropdown.md (hunk 2 above)
Comment (s-pace) [reply docs/src/dropdown.md]: Thx

File: docs/src/faq.md (hunk 3 above)
Comment (s-pace) [reply docs/src/faq.md]: We want to avoid using this wording but you are right. Will not do it

File: docs/src/faq.md (hunk 4 above)
Comment (s-pace) [reply docs/src/faq.md]: Thx

File: docs/src/faq.md (hunk 6 above)
Comment (s-pace) [reply docs/src/faq.md]: Thanks

File: docs/src/faq.md (hunk 4 above)
Comment (s-pace) [reply docs/src/faq.md]: We do not have a specific link. Using the email address but this is somewhere else. Thanks

File: docs/src/faq.md (hunk 3 above)
Comment (Haroenv) [reply docs/src/faq.md]: I don't understand your comment, but you're more aware of the word :)

Comment (s-pace) [reply docs/src/faq.md]: We do not want to over promote "scraper" since we have a dedicated crawler

Comment (s-pace) [reply docs/src/faq.md]: Please approve if you do not need to change anything please

Comment (Haroenv) [reply docs/src/faq.md]: I thought it was the other way round (removing the word "crawler" here)

File: docs/src/dropdown.md [hunk 7]
```diff
@@ -35,6 +35,9 @@ config.
 </script>
 ```
 
+This snippet needs to be integrated to every page that integrates the dropdown UI
+dropdown UI.
```
Comment (francoischalifour) [inline docs/src/dropdown.md]: This is there twice.

File: docs/src/faq.md (hunk 3 above)
Comment (francoischalifour) [reply docs/src/faq.md]: I understood the same as @Haroenv. We don't want to use the word "crawler" anymore.

**Pull request merged**